### ISSUE TITLE
hmmprofile syntax brokenn in cazyhmmer.out

### ISSUE THREAD

**Issue by Neato-Nick** [linnabrown/run_dbcan#39]
Hi, I've been using run_dbcan.py on the command line running all three tools. I've noticed that in the hmmer output, the syntax for certain hmm profiles is not valid.

In general, I've noticed the output pattern is 

"${superfamily}${family}_${subfamily}.hmm"

**Specifically for GT2**, this syntax is broken. I have not noticed this with any other cazyme in my results. Instead, I get the following results and never only "GT2.hmm" or a subfamily like "GT2_2.hmm":

GT2_Glycos_transf_2.hmm
GT2_Glyco_trans_2_3.hmm
GT2_Chitin_synth_1.hmm
GT2_Chitin_synth_2.hmm

There could be other families this affects, since this subset is limited to whatever proteins are from my organism.

I'm trying to avoid using the webserver, looks like it's got a backlog of jobs to get through, but this makes it very hard to parse the results unless I just throw out everything in the GT2 family.

**Comment by yinlabniu** [comment]
Nick,

Thanks for catching this. In 2018, we used 8 Pfam HMMs to replace dbCAN GT2. These 8 Pfam HMMs are:

GT2_Cellulose_synt
GT2_Chitin_synth_1
GT2_Chitin_synth_2
GT2_Glycos_transf_2
GT2_Glyco_tranf_2_2
GT2_Glyco_tranf_2_3
GT2_Glyco_tranf_2_4
GT2_Glyco_tranf_2_5
GT2_Glyco_trans_2_3

This was announced in http://bcb.unl.edu/dbCAN2/download/Databases/dbCAN-old@UGA/readme.txt.

This may have disrupted the codes that parse the model names. This should be an easy fix, and we will look into it.

Yanbin

________________________________
